docs/api/explain-product-attributes.md

```diff
@@ -117,10 +121,101 @@ In order not to break existing clients, this attribute is listed in the response
 
 The response above indicates that the **Unwanted ingredients** attribute needs a `attribute_unwanted_ingredients_tags` parameter which is a list of comma separated canonical ingredients tags (e.g. "en:garlic,en:kiwi")
 
+##### Sending unwanted/banned ingredients in requests
+
 This parameter can be sent in product read and search requests in 2 ways:
 - as a cookie with the name `attribute_unwanted_ingredients_tags`: this is used in particular on the website so that we do not have URLs with an extra query parameter.
 - as a query parameter in the URL (e.g. ?attribute_unwanted_ingredients_tags=en:garlic,en:kiwi)
 
+##### Getting ingredient suggestions
```

Comment:
All the things below are interesting, but this doc is about explaining product attributes (in general) succinctly. Explain docs should stay on topic, and all the detailed examples, guidance etc. can go into a separate tutorial "How to implement the unwanted ingredients attribute".